File: source/EventManagerBackend (Rider)/EventManagerBackend/Model/Event.cs
using System.Text.Json.Serialization;

namespace EventManagerBackend.Model
{
    public class Event
    {
        [JsonPropertyName("id")]
        public int? Id { get; set; }
        [JsonPropertyName("name")]
        public string Name { get; set; }
        [JsonPropertyName("hidden")]
        public bool Hidden { get; set; }
        [JsonPropertyName("start")]
        public long Start { get; set; }
        [JsonPropertyName("end")]
        public long End { get; set; }
        [JsonPropertyName("equipments")]
        public Equipment[] Equipments { get; set; }
    }
}

File: source/EventManager (AndroidStudio)/app/src/main/java/com/SovietHouseholdAppliances/EventManager/model/Event.java
package com.SovietHouseholdAppliances.EventManager.model;

import androidx.annotation.NonNull;

public class Event
{
    public Integer id;
    public String name;
    public boolean hidden;
    public long start;
    public long end;
    public Equipment[] equipments;

    public Event(Integer id, String name, boolean hidden, long start, long end, Equipment[] equipments)
    {
        this.id = id;
        this.name = name;
        this.hidden = hidden;
        this.start = start;
        this.end = end;
        this.equipments = equipments;
    }

    @Override
    public boolean equals(Object obj) {
        if (!(obj instanceof Event))
            return false;
        if (id == null || ((Event) obj).id == null)
            return false;
        return id.equals(((Event) obj).id);
    }

    @NonNull
    @Override
    public String toString() {
        return name;
    }
}